Getting templates from the system path doesn't work

Listing works, but getting and and therefore creating jobs that use them doesn't.

```
$ krontab get template pangeo
panic: open /home/jovyan/.config/krontab/krontab/templates/pangeo.yaml: no such file or directory

goroutine 1 [running]:
github.com/jacobtomlinson/krontab/template.GetTemplate(0x7fffaa2c53f3, 0x6, 0xc000161cc8, 0x598075, 0x1c52620, 0x59d4f0)
        /home/travis/gopath/src/github.com/jacobtomlinson/krontab/template/template.go:150 +0x307
github.com/jacobtomlinson/krontab/cmd.glob..func5(0x1c52620, 0xc000283f50, 0x1, 0x1)
        /home/travis/gopath/src/github.com/jacobtomlinson/krontab/cmd/get.go:25 +0x49
github.com/spf13/cobra.(*Command).execute(0x1c52620, 0xc000283f30, 0x1, 0x1, 0x1c52620, 0xc000283f30)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0x1c53460, 0x0, 0x1c53b80, 0xc000161f58)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0x1c53460, 0x407460, 0xc0000ae058)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/jacobtomlinson/krontab/cmd.Execute()
        /home/travis/gopath/src/github.com/jacobtomlinson/krontab/cmd/root.go:44 +0x2d
main.main()
        /home/travis/gopath/src/github.com/jacobtomlinson/krontab/main.go:8 +0x20
```

--- a/template/template.go
+++ b/template/template.go
@@ -96,6 +96,7 @@ func EditTemplate(template string) error {
 			templatePath := filepath.Join(templateDir, template+".yaml")
 			if _, err := os.Stat(templatePath); err == nil {
 				path = templatePath
+				break
 			}
 		}
 		if path == "" {
@@ -136,6 +137,7 @@ func DeleteTemplate(template string) error {
 			templatePath := filepath.Join(templateDir, template+".yaml")
 			if _, err := os.Stat(templatePath); err == nil {
 				path = templatePath
+				break
 			}
 		}
 		if path == "" {
@@ -161,6 +163,7 @@ func GetTemplate(template string) (string, error) {
 			templatePath := filepath.Join(templateDir, template+".yaml")
 			if _, err := os.Stat(templatePath); err == nil {
 				path = templatePath
+				break
 			}
 		}
 		if path == "" {

--- a/template/template.go
+++ b/template/template.go
@@ -91,9 +91,15 @@ func EditTemplate(template string) error {
 		return errors.New("you cannot edit the default template")
 	}
 	if IsTemplate(template) {
-		path := filepath.Join(config.TemplateDirs[0], template+".yaml")
-		if _, err := os.Stat(path); os.IsNotExist(err) {
-			path = filepath.Join(config.TemplateDirs[1], template+".yaml")
+		var path string
+		for _, templateDir := range config.TemplateDirs {
+			templatePath := filepath.Join(templateDir, template+".yaml")
+			if _, err := os.Stat(templatePath); err == nil {
+				path = templatePath
+			}
+		}
+		if path == "" {
+			panic(fmt.Errorf("Cannot find template %s", template))
 		}
 		input.UserEdit(path)
 		// TODO Validate is valid CronJob template
@@ -125,9 +131,15 @@ func DeleteTemplate(template string) error {
 		return errors.New("you cannot delete the default template")
 	}
 	if IsTemplate(template) {
-		path := filepath.Join(config.TemplateDirs[0], template+".yaml")
-		if _, err := os.Stat(path); os.IsNotExist(err) {
-			path = filepath.Join(config.TemplateDirs[1], template+".yaml")
+		var path string
+		for _, templateDir := range config.TemplateDirs {
+			templatePath := filepath.Join(templateDir, template+".yaml")
+			if _, err := os.Stat(templatePath); err == nil {
+				path = templatePath
+			}
+		}
+		if path == "" {
+			panic(fmt.Errorf("Cannot find template %s", template))
 		}
 		err := os.Remove(path)
 		if err != nil {
@@ -144,7 +156,16 @@ func DeleteTemplate(template string) error {
 // GetTemplate opens a template and reads as a string
 func GetTemplate(template string) (string, error) {
 	if IsTemplate(template) {
-		path := filepath.Join(config.TemplateDirs[0], template+".yaml")
+		var path string
+		for _, templateDir := range config.TemplateDirs {
+			templatePath := filepath.Join(templateDir, template+".yaml")
+			if _, err := os.Stat(templatePath); err == nil {
+				path = templatePath
+			}
+		}
+		if path == "" {
+			panic(fmt.Errorf("Cannot find template %s", template))
+		}
 		dat, err := ioutil.ReadFile(path)
 		if err != nil {
 			panic(err)

